Dockerfile

```diff
@@ -0,0 +1,49 @@
+FROM python:2.7.8-wheezy
+MAINTAINER James Swanick "swanijam@gmail.com"
+
+RUN curl http://python-distribute.org/distribute_setup.py | `which python`
+RUN curl https://raw.github.com/pypa/pip/master/contrib/get-pip.py | `which python`
+
+RUN apt-get update -qq
```

Comment:
Other than `unzip`, try removing all of these extract packages to see if what is included in this image suffices. Add packages only where needed.